content/en/docs/platforms/client-apps/android/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Android
+description: >-
+  Using OpenTelemetry in apps running on Android platforms
```

Comment:
Let's make these complete sentences. They appear at the top of each page, as you can see in the preview.

```suggestion
  Use OpenTelemetry in apps running on Android platforms
```